nameko/timer.py

```diff
@@ -33,8 +35,11 @@ def tick(self):
 
         """
         self.interval = interval
+        self.eager = eager
         self.should_stop = Event()
+        self.worker_complete = Event()
         self.gt = None
+        super(Timer, self).__init__()
```

Comment:
Added this while I was at it.  If this isn't added and there's an exception in the method that uses the timer endpoint there's an exception in the service container caused by a missing `expected_exceptions` endpoint.  The `test_timer_error` test below tests this.